rust/sealevel/abacus-core/src/types/message.rs

```diff
@@ -0,0 +1,210 @@
+use crate::types::H256;
+use sha3::{Digest, Keccak256};
+use sha3::digest::Update;
+
```

Comment:
Recommend taking a look at the new types in `main` (which includes v2 now) and copying those over, as I think some are stale